source/MaterialXTest/Runtime.cpp

```diff
@@ -1428,6 +1428,22 @@ TEST_CASE("Runtime: materials", "[runtime]")
     api->loadLibrary(STDLIB);
     api->loadLibrary(PBRLIB);
     api->loadLibrary(BXDFLIB);
+
+    // Set and test search paths
+    api->clearSearchPath();
```

Comment:
We should be careful with search paths as api never get's cleared while it is in existence.